app/views/pages/embedded.html

```diff
@@ -0,0 +1,74 @@
+{% extends "layouts/base.html" %}
+
+{% block content %}
+{% raw %}
+<style>
+  .alert {
+    display: none; !important;
+  }
+</style>
+
+<div>
+  <div class="waiter text-center"
+    ng-if="(state.isPackageLoading || state.isStarting)">
+    <i class="fa fa-spinner fa-pulse fa-4x"></i>
+    <span>Loading...</span>
+  </div>
+  <div ng-if="!state.isStarting">
+    <div class="col-md-12 col-xs-12 charts">
+      <babbage
+              ng-if="!state.flag.renderingCharts && currentTab == 'treemap'"
```

Comment:
@pwalsh We have several tabs in non-embedded app. This variable ('currentTab') has two goals :
1. show active tab
2. drops all non-active charts from DOM. Because, if we don't drop charts from hidden tabs our browser will work very slow.